Update mavenTestAndValidate.yml

team172011 opened the pull request:
add xvfb-run for processing ui test cases

nimo23 commented:
@team172011 I get this error because of this?:

![screenshot](https://user-images.githubusercontent.com/3045549/105626633-11349800-5e31-11eb-8044-76d9aa05bf96.png)

team172011 commented:
@nimo23 yes, I changed the tests from travis to githubactions and after the first attempt they failed because of the graphical tests in ta4j-example. But I already updated this action, if you click on the "checks" tab you see everything is fine
